Runtime/AvatarCreator/Scripts/Managers/AvatarManager.cs

```diff
@@ -373,5 +384,13 @@ public void Dispose()
         {
             ctxSource?.Cancel();
         }
+
+        private void ThrowErrIfCancellationException(Exception e)
+        {
+            if (e.Message == TaskExtensions.ON_REQUEST_CANCELLED_MESSAGE)
```

Comment:
you can check for OperationCanceledException here instead of the string comparison

Reply:
response exceptions are regular exceptions, not OperationCanceledExceptions
https://github.com/readyplayerme/rpm-unity-sdk-core/blob/main/Runtime/AvatarCreator/Scripts/Extensions/ResponseExtensions.cs